sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorHandlerProvider.java

```diff
@@ -34,9 +39,24 @@ public class ReactorHandlerProvider {
      * @param provider The provider that creates and manages {@link Reactor} instances.
      *
      * @throws NullPointerException If {@code provider} is {@code null}.
+     * @deprecated use {@link ReactorHandlerProvider#ReactorHandlerProvider(ReactorProvider, Meter)} instead.
      */
+    @Deprecated
     public ReactorHandlerProvider(ReactorProvider provider) {
+        this(provider, null);
+    }
+
+    /**
+     * Creates a new instance with the reactor provider to handle {@link ReactorDispatcher ReactorDispatchers} to its
+     * generated handlers.
+     *
+     * @param provider The provider that creates and manages {@link Reactor} instances.
+     *
```

Comment:
Missing `@param` for meter (not that it is critical seeing as this is non-public API)